styles.css

```diff
@@ -0,0 +1,104 @@
+* {
+  box-sizing: border-box;
+  /* background-color: #386acd67; */
+}
+div {
+  display: block;
+}
+body {
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 16px;
+  font-weight: 400;
+  line-height: 1.4;
+  color: #ffffff;
+  margin: 0;
+}
+body::before {
+  content: "";
+  position: fixed;
+  top: 0;
+  left: 0;
+  height: 100%;
+  width: 100%;
+  z-index: -1;
+  background: var(--color-darkblue);
+  background-image: linear-gradient(
+      115deg,
+      rgba(58, 58, 158, 0.8),
+      rgba(136, 136, 206, 0.7)
+    ),
+    url(https://cdn.freecodecamp.org/testable-projects-fcc/images/survey-form-background.jpeg);
+  background-size: cover;
+  background-repeat: no-repeat;
+  background-position: center;
+}
+h1 {
+  font-weight: 400;
+  line-height: 1.2;
+  font-size: 40px;
+}
+p {
+  font-size: 18px;
+}
+h1,
+p {
+  margin-top: 0;
+  margin-bottom: 0.5rem;
+}
+label {
+  display: flex;
+  align-items: center;
+  font-size: 1.125rem;
+}
+.header {
+  display: block;
+  padding: 0 0.625rem;
+  margin-bottom: 1.875rem;
+  text-align: center;
+  font-size: 32px;
+}
+form {
+  padding: 2.5rem;
+  background: #214b22;
+  border-radius: 0.25rem;
+  display: block;
+  margin-top: 0em;
+  margin-left: 1.2em;
+  margin-right: 1.2em;
+}
+.form-type {
+  margin: 0 auto 1.25rem auto;
+  padding: 0.25rem;
+  margin-bottom: 0.5rem;
+}
+.form-control {
+  width: 100%;
+  height: 2.375rem;
+  padding: 8px 10px;
+  color: #1f211f;
+  border-radius: 8px;
+  border: solid 1px #000000;
+}
+.input-textarea {
+  min-height: 120px;
+  width: 100%;
+  padding: 8px 10px;
+}
+.submit-button {
+  display: block;
+  width: 100%;
+  background: #13b6cb;
+  border-radius: 8px;
+  cursor: pointer;
+  height: 3rem;
+  color: #ffffff;
+}
+.container {
+  /* width:100%; */
```

Comment:
Good job! Please do not leave unused codes as comments on your codebase. they don't make your code look neat and easily readable.